docs/ref/utils.txt

```diff
@@ -832,6 +832,11 @@ appropriate entities.
 
     Can be called multiple times on a single string.
 
+    .. versionchanged:: 1.11
```

Comment:
Please make this annotation "self-contained" as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features and move it to the bottom of this section.